### PR TITLE
Fix for octobercms/october#3176

### PR DESCRIPTION
**Pull request by GinoPane**
A small update to handle server's timeout on client side.

image

### CODE REVIEW

File: install_files/partials/progress/fail.htm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="callout callout-danger">
     {{#reason}}
-        <h4>{{reason}}</h4>
+        <h4>{{{reason}}}</h4>
```
Comment (LukeTowers) [inline install_files/partials/progress/fail.htm]: What's the extra curly braces here for?

**Comment by LukeTowers** [comment]
Thanks for your work on this @GinoPane!

**Comment by GinoPane** [comment]
No problem! Thanks for reviewing this.
Would be glad to participate more when I could.

**Comment by LukeTowers** [comment]
@GinoPane there's lots of open issues on octobercms/october, the docs could always use improvements, any open PRs on octobercms/october, octobercms/library, octobercms/install, octobercms/docs could be reviewed; and there's also an unending stream of user's questions on the forums and stackoverflow :). Not to mention all of the RainLab plugins as well that we maintain.

Have at er! 😂

**Pull request merged**